### ISSUE TITLE
Confusion in using Pretrained weights.

### ISSUE THREAD

**Issue by vikas-baghel** [chensong1995/HybridPose#30]
Hi, I wanted to see how the model is working so I thought I would download the pre-trained weights and give some images as input and see the 3D bounding box as output. But I can see that there are different weights for different classes. Why there are different weights? I am new in this field. As far as I know, we require only one pre-trained weight for all the classes? Please help me with this. Sorry if I am missing some obvious things.

**Comment by chensong1995** [comment]
Hello Vikas,

Thanks for you interest in our research!

We follow the convention in PVNet and train a new network for every object class. However, it is easy to modify the model and train one single network for all object classes. You will need to change the network output to predict multi-class semantic segmentation rather than binary semantic segmentation masks. You can then extract intermediate representations from each object in the image, and feed them to the pose regression module described in the paper.

I hope this helps,
Chen
